blocks/core/_lib/ff-tabs/_ff-tabs-core.js

```diff
@@ -113,16 +122,61 @@ function getSwapStatesMethod($root, options) {
                 addCompleteClasses($lastLinks, $lastContent);
             }
         }
+    }
+
+    function handleClick(e) {
+        e.preventDefault();
+        /*jshint validthis:true */
+        setState(this);
+    }
 
-    };
-}
 
-function setClickHandler($root, options) {
-    var linkSel = '[' + options.linkSelBase + ']';
-    $root.on('click', linkSel, getSwapStatesMethod($root, options));
+    function testBounds(value, length) {
+        if (value < 0) {
+            return length - 1;
+        } else if (value >= length) {
+            return 0;
+        } else return value;
+    }
+
+    function setActiveTab(index) {
```

Comment:
Sorry, which part do you mean?
We do need methods that simply advance the tabs, for the set-a-task form step flow, so I do think we need next and (last/previous)